Platon/backend/app/follow_system/views.py

```diff
@@ -1,10 +1,250 @@
+                db.session.add(new_follow_entry)
+                db.session.delete(follow_request)
+                db.session.commit()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            return make_response(jsonify({'msg': 'Follow Request is succesfully accepted'}), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Input Format Error'}), 400)
+
+
+# login_required decorator will be added.
+class RejectFollowRequestAPI(Resource):
+
+    @api.doc(responses={200: 'Follow Request is rejected successfully',
+                        400: 'Input Format Error',
+                        401: 'Current user is unauthorized to reject the Follow Request',
+                        404: 'Follow Request not found',
+                        500: 'Database Connection Error'})
+    @api.expect(reject_follow_requests_parser)
+    @login_required
+    def post(user_id, self):
+        '''
+            Takes the follower_id and following_id as inputs and rejects the corresponding Follow Request.
+        '''
+        form = AcceptFollowRequestsForm(request.form)
+        if form.validate():
+
+            # Current user should be the following.
+            if user_id != form.following_id.data:
+                return make_response(jsonify({'error': 'Unauthorized'}), 401)
+
+            # Check if the given FollowRequest instance exists.
+            try:
+                followSearch = FollowRequests.query.filter(FollowRequests.following_id == form.following_id.data,
+                                                           FollowRequests.follower_id == form.follower_id.data).all()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            if len(followSearch) == 0:
+                return make_response(jsonify({'error': 'Follow Request not found'}), 404)
+
+            # If it exists: Delete FollowRequest instance.
+            follow_request = followSearch[0]
+            try:
+                db.session.delete(follow_request)
+                db.session.commit()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Problem'}), 500)
+
+            return make_response(jsonify({'msg': 'Follow Request is succesfully accepted'}), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Bad Request'}), 400)
 
 
 def register_resources(api):
-    api.add_resource(FollowAPI,"/follow")
\ No newline at end of file
+    api.add_resource(GetFollowingsAPI, "/follow/followings")
+    api.add_resource(GetFollowersAPI, "/follow/followers")
+    api.add_resource(GetFollowRequestsAPI, "/follow/get_follow_requests")
+    api.add_resource(SendFollowRequestAPI, "/follow/send_follow_requests")
+    api.add_resource(AcceptFollowRequestAPI, "/follow/accept_follow_requests")
+    api.add_resource(RejectFollowRequestAPI, "/follow/reject_follow_requests")
```

Comment:
Sorry for the late response but I also learned this feature today. Can you use namespaces to determine routes? Because it creates much better API documentation.

Reply:
I agree with you. I've changed it with the last commit. Thanks